### PR TITLE
Certificates, EPS configuration

### PR DESCRIPTION
**Pull request by lucky-lusa**

### CODE REVIEW

File: connect_your_app_to_IRIS/technical_details/app_onboarding.md [hunk 1]
```diff
@@ -27,13 +27,26 @@ Please find below and overview of the IRIS system and its actors as well as the
...
 
-First, the EPS is installed and configured locally. Then messages can be sent to and received from IRIS via the EPS.
+Generate your certificate signing request 
 
-The EPS server can be cloned and created from [https://github.com/iris-gateway/eps](https://github.com/iris-gateway/eps). We will later also provide an executable, versioned binary for all common environments.
+    openssl genrsa -out "${cert}.key" 2048;
+  	openssl rsa -in "${cert}.key" -pubout -out "${cert}.pub";
+  	openssl req -new -sha256 -key "${cert}.key" -subj "/C=${C}/ST=${ST}/L=${L}/O=${O}/OU=${OU}/CN=${cert}" -addext "subjectAltName = DNS:${cert},DNS:*.${cert}.local" -out "${cert}.csr";
 
+with your information in subject and your app name as ${cert}. Please use your app name as CN (for example CN=smartmeeting). Don't use spaces.
```
Comment (hey-johnnypark) [inline connect_your_app_to_IRIS/technical_details/app_onboarding.md:37]: I suggest to provide a full example from Smart Meeting. That will make it really clear what we expect. 

```
O="COSYNUS GmbH"
ST="Europaplatz 5"
L="64293 Darmstadt"
C="DE"
OU="IT"
CN="smartmeeting"
# using less than 1024 here will result in a TLS handshake failure in Go
# using less than 2048 will cause e.g. 'curl' to complain that the ciper is too weak
LEN="2048"
```

File: connect_your_app_to_IRIS/technical_details/app_onboarding.md [hunk 2]
```diff
@@ -27,13 +27,26 @@ Please find below and overview of the IRIS system and its actors as well as the
 ## Request a certificate
 
 ToDo: Describe process how to contact the rollout team and what information is expected in the CSR.
```
Comment (hey-johnnypark) [inline connect_your_app_to_IRIS/technical_details/app_onboarding.md:29]: Can be removed. You explain that they need to send an E-Mail.

File: connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# we don't want to check in test or dev certificates
```
Comment (hey-johnnypark) [inline connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/.gitignore:1]: Did you forget something here?

File: connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```
Comment (hey-johnnypark) [inline connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt:1]: Should this be checked in?

Comment (hey-johnnypark) [reply connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt:1]: Ah I think this is the cert that you trust on Staging environment right?

File: connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/roles/yourapp/001_default.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      bind_address: "localhost:4445"
+      tls:
+        ca_certificate_file: "$DIR/../../certs/root.crt"
+        certificate_file: "$DIR/../../certs/[yourapp].crt"
+        key_file: "$DIR/../../certs/[yourapp].key"
+  - name: main JSON-RPC server # accepts incoming JSONRPC connections to deliver and receive messages
+    type: jsonrpc_server
+    settings:
+      bind_address: "localhost:5556"
+      cors:
```
Comment (hey-johnnypark) [inline connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/roles/yourapp/001_default.yml:29]: Can CORS be removed?

**Review by hey-johnnypark — COMMENTED** [review]
I left some comments. Well done.

File: connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt (hunk 4 above)
Comment (lucky-lusa) [reply connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/certs/staging-root.crt:1]: Yes, that's the trusted cert for staging. Can therefore be stored in the respository.

File: connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/roles/yourapp/001_default.yml (hunk 5 above)
Comment (lucky-lusa) [reply connect_your_app_to_IRIS/technical_details/eps-config/settings/staging/roles/yourapp/001_default.yml:29]: Yes, not necessary in that environment.

**Pull request merged**